#Vehicle maximum accelaration
max_accelaration: 5.0

#Vehicle maximum deceleration
max_deceleration: 8.0

#The weight of the cost of feasibility
weight_of_feasibility: 1

#The weight of the cost of safety
weight_of_safety: 1

#The weight of the cost of comfort
weight_of_comfort: 1

#The weight of the cost of efficiency
weight_of_efficiency: 1

#The weight of the cost of fuel
weight_of_fuel: 1

# This needs to be dynamic based on road environment.
# Need to create a story for this feature.
#Speed limit
#unit: m/s
speed_limit: 27

#Speed buffer
#unit: m/s
speed_bufffer: 25


